charts/argo-workflows/values.yaml

```diff
@@ -161,9 +161,13 @@ controller:
   # Only valid for 2.7+
   ## See more: https://argoproj.github.io/argo-workflows/default-workflow-specs/
   workflowDefaults: {}
-  #  spec:
-  #    ttlStrategy:
-  #      secondsAfterCompletion: 84600
+  #   spec:
```

Comment:
I apologise for unsollicited review comments - there appear to be three spaces here. I appreciate the effort.

Reply:
Thank you for your comment(always welcome!), but it is intended. My vsCode inserts `# ` when comment out.
There are some parts in `values.yaml` commented out as the same way.